invoices/invoice_expiry_watcher.go

```diff
@@ -181,6 +181,24 @@ func (ew *InvoiceExpiryWatcher) cancelNextExpiredInvoice() {
 	}
 }
 
+// pushInvoices adds invoices to be expired to their relevant queue.
+func (ew *InvoiceExpiryWatcher) pushInvoices(invoices []queue.PriorityQueueItem) {
```

Comment:
nit: using the `PirorityQueueItem` makes the code slightly less readable but it's probably better this way since we need to scan through all invoices at start.

Reply:
Yeah also thought about this one a bit. Since this will be used for another type of expiry, I though this would be simpler. The alternative is to have `pushTsExpiry` and `pushHeightExpiry` further down the line, but then the caller needs to handle some logic. 

Going to add a vanity `invoiceExpiry` interface for the different expiry types, it's one extra line and I think it'll help with readability.